akka-remote/src/main/scala/akka/remote/RemoteMetricsExtension.scala

```diff
@@ -56,35 +59,48 @@ private[akka] class RemoteMetricsOff extends RemoteMetrics {
  */
 private[akka] class RemoteMetricsOn(system: ExtendedActorSystem) extends RemoteMetrics {
 
-  private val logFrameSizeExceeding: Int = system.settings.config.getBytes(
-    "akka.remote.log-frame-size-exceeding").toInt
   private val log = Logging(system, this.getClass)
+
+  private val logFrameSizeExceeding: Try[Int] = Try(
+    system.settings.config.getBytes("akka.remote.log-frame-size-exceeding").toInt
+  )
```

Comment:
A try-catch would be better than a `Try` in this case, this will cause even message to allocate an extra function and go through an extra dispatch to that function.

Reply:
Hey there :). Not perfectly sure what you mean. "... will cause even message ..." Do you mean every message? I'd expect this code to be executed only once for the first message, am I mistaken here?
Or are you merely referring to the `logFrameSizeExceeding.failed.foreach` that could be avoided if using a try-catch. In that case I'd suggest to make it a `logFrameSizeExceeding: Option[Int]`.